packages/css/src/utils/css-property.ts

```diff
@@ -23,13 +26,13 @@ const unitless = {
   gridColumnEnd: true,
   gridColumnSpan: true,
   gridColumnStart: true,
-  fontWeight: true,
```

Comment:
why has this changed? Is it incorrect?

Just thinking about the implications of bumping a change like this for something that's heavily used

Reply:
It was moved to line 19 to keep the document pseudo-alphabetical 😉

Reply:
oops cheers for the explanation